test/android/patches/makeSettingsPatch.spec.js

```diff
@@ -5,17 +5,19 @@ const expect = chai.expect;
 const makeSettingsPatch = require('../../../src/android/patches/makeSettingsPatch');
 
 const name = 'test';
-const projectConfig = {
-  sourceDir: '/home/project/android/app',
-  settingsGradlePath: '/home/project/android/settings.gradle',
-};
-const dependencyConfig = { sourceDir: `/home/project/node_modules/${name}/android` };
+const projectConfig = { settingsGradlePath: 'some/path/android' };
```

Comment:
SettingsGradlePath is `some/path/android/settings.gradle` - that's sent by rnpm link (as in previous unchanged version)